tidb-cloud/vector-search-index.md

```diff
@@ -133,6 +133,34 @@ LIMIT 5;
 
 See [Table Partitioning](/partitioned-table.md) for more information.
 
+## View index build progress
+
+Unlike other indexes, vector indexer is asynchronous. As a result, after inserted bulk data, vector index may be not available yet. This will not affect correctness or consistency -- you can always perform vector search at any time and get complete results. However, performance will be suboptimal until vector index is built. In this case, you can view index build progress by querying the `INFORMATION_SCHEMA`.`TIFLASH_INDEXES` table:
+
+```sql
+[tidb]> SELECT * FROM INFORMATION_SCHEMA.TIFLASH_INDEXES;
++---------------+------------+----------------+----------+--------------------+-------------+-----------+------------+---------------------+-------------------------+--------------------+------------------------+------------------+
+| TIDB_DATABASE | TIDB_TABLE | TIDB_PARTITION | TABLE_ID | BELONGING_TABLE_ID | COLUMN_NAME | COLUMN_ID | INDEX_KIND | ROWS_STABLE_INDEXED | ROWS_STABLE_NOT_INDEXED | ROWS_DELTA_INDEXED | ROWS_DELTA_NOT_INDEXED | TIFLASH_INSTANCE |
++---------------+------------+----------------+----------+--------------------+-------------+-----------+------------+---------------------+-------------------------+--------------------+------------------------+------------------+
+| test          | sample     | NULL           |      106 |                 -1 | vec         |         2 | HNSW       |                   0 |                   13000 |                  0 |                   2000 | store-6ba728d2   |
+| test          | sample     | NULL           |      106 |                 -1 | vec         |         2 | HNSW       |               10500 |                       0 |                  0 |                   4500 | store-7000164f   |
++---------------+------------+----------------+----------+--------------------+-------------+-----------+------------+---------------------+-------------------------+--------------------+------------------------+------------------+
+```
+
+- `ROWS_STABLE_INDEXED` and `ROWS_STABLE_NOT_INDEXED` columns show index build progress. When `ROWS_STABLE_NOT_INDEXED` becomes 0, index build is complete.
+
+  As a reference, indexing 500 MiB vector dataset may take up to 20 minutes. Currently it is not possible to adjust indexer priority or speed. The indexer may run in parallel for multiple tables.
+
+- `ROWS_DELTA_NOT_INDEXED` column shows how many rows are in Delta layer. Delta layer stores _recently_ inserted or updated rows. Delta layer is periodically merged into Stable layer according to the write workload (which is called Compaction).
+
+  Delta layer is always not indexed. Sometimes you may want to achieve best performance by forcing the merge of Delta layer into Stable layer, so that all data can be indexed:
```

Comment:
```suggestion
  The Delta layer is always not indexed. To achieve optimal performance, you can force the merge of the Delta layer into the Stable layer so that all data can be indexed:
```